node/core/pvf/src/prepare/worker.rs

```diff
@@ -169,6 +166,77 @@ pub async fn start_work(
 	.await
 }
 
+/// Handles receiving response bytes on the host from the child.
```

Comment:
This is a little deceiving. It doesn't handle receiving, it decoded an already received bytes.